web/src/elements/buttons/ModalButton.ts

```diff
@@ -1,7 +1,7 @@
 import { AKElement } from "@goauthentik/elements/Base";
 import { PFSize } from "@goauthentik/elements/Spinner";
 
```

Comment:
A small thing, but using `nothing` instead of "html``" allows lit better control over  rendering and reduces the number of actual renders of the page.

Reply:
we might want to make that change everywhere later

Reply:
I do it as I touch files.  Things that don't need to be updated or fixed ... don't need to be updated or fixed.  Yet.  :-)